src/app/Pages/Blog.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
+import {
+   PageSection, Bullseye, Text, TextContent, TextVariants,
+  Card, CardTitle, CardBody, CardFooter
+} from '@patternfly/react-core'
+import { useDocumentTitle } from '@app/utils/useDocumentTitle'
+import Footer from '@app/components/Footer'
+import post11png from '@app/bgimages/post1-1.png'
+
+const Blog: React.FunctionComponent<{title: string}> = ({title}) => {
+
+  useDocumentTitle(title)
+  return (
+    <div>
+      <PageSection isFilled={true}>
+      <Bullseye>
+        <Card
+          style={{
+            width: 800,
+            display: 'flex',
+            justifyContent: 'center',
+          }}>
+          <CardTitle>
+            <p>The new Cloud Image Directory is now available</p>
+            <Text component={TextVariants.small}>Author: The Cloud Experience Team</Text>
+          </CardTitle>
+          <CardBody isFilled={false}>
+            <Bullseye>
+              <img src={post11png}
+              style={{
+                display: 'flex',
+                justifyContent: 'center',
+              }}></img>
+            </Bullseye>
+            <TextContent>
+              <Text component={TextVariants.p}>
+                Our Cloud Experience team is excited to announce the availability of the first version of our Cloud Image Directory.
+                This tool is in an experimental stage, however we have decided to release this early access version to our community
+                and stay true to our open culture and engineering principles and invite you to join us in this journey.
+              </Text>
+
+              <Text component={TextVariants.p}>
+                Based on community feedback, we realized that many users today struggle to find out when and where new images are available,
+                because they are neither actively promoted on cloud marketplaces, nor easily accessible on other available platforms.
+                In addition, research suggests that experienced users will benefit from searching for particular versions of cloud images,
+                like specific older versions. Finally, you can use this tool to try out your preferred images before considering other requirements,
+                such as specific subscriptions and licenses.
+              </Text>
+
+              <Text component={TextVariants.h3}>
+                What this new tool is about
+              </Text>
+
+              <Text component={TextVariants.p}>
+                Our  new tool provides easy access to all Linux cloud images.  Our vision is to help users search and quickly find any publicly
+                available cloud image so they can conveniently deploy it to their preferred environment.
+                It provides an easy one-step dashboard that points you to the latest cloud image release on the marketplace of your choice.
```

Comment:
Please rewrite as follows:

It provides a one-step dashboard that points to the latest cloud image release on the marketplace of your choice. If you are looking for a single-click spin-up for the newest version of your favorite distribution, you’re set. If you are looking for a cloud image that comes with a specific version or feature, use the provider-specific browser to search, find, and select the image from a variety of offers.